modules/machineset-yaml-osp-sr-iov-port-security.adoc

```diff
@@ -94,40 +94,4 @@ Trunking is enabled for ports that are created by entries in the networks and su
 You can enable trunking for each port.
 
 Optionally, you can add tags to ports as part of their `tags` lists.
-====
-
-If your cluster uses Kuryr and the {rh-openstack} SR-IOV network has port security disabled, the primary port for compute machines must have:
-
-* The value of the `spec.template.spec.providerSpec.value.networks.portSecurityEnabled` parameter set to `false`.
-
-* For each subnet, the value of the `spec.template.spec.providerSpec.value.networks.subnets.portSecurityEnabled` parameter set to `false`.
-
-* The value of `spec.template.spec.providerSpec.value.securityGroups` set to empty: `[]`.
-
-.An example section of a compute machine set for a cluster on Kuryr that uses SR-IOV and has port security disabled
-[source,yaml]
-----
-...
-          networks:
-            - subnets:
-              - uuid: <machines_subnet_UUID>
-                portSecurityEnabled: false
-              portSecurityEnabled: false
-          securityGroups: []
-...
-----
-
-In that case, you can apply the compute security group to the primary VM interface after the VM is created. For example, from a command line:
-[source,terminal]
-----
-$ openstack port set --enable-port-security --security-group <infrastructure_id>-<node_role> <main_port_ID>
-----
-
-[IMPORTANT]
-====
-After you deploy compute machines that are SR-IOV-capable, you must label them as such. For example, from a command line, enter:
-[source,terminal]
-----
-$ oc label node <NODE_NAME> feature.node.kubernetes.io/network-sriov.capable="true"
-----
 ====
```

Comment:
Delete this clsoing delimiter

Reply:
This line is required to close the NOTE that is opened on line 91